pkg/users/users.go

```diff
@@ -0,0 +1,24 @@
+// Package users provides a client wrapping the WorkOS User Management API.
+package users
+
+import "context"
+
+var (
+	// DefaultClient is the client used by GetAuthorizationURL, GetProfileAndToken and
+	// Login functions.
+	DefaultClient = NewClient("")
+)
+
+// Configure configures the default client that is used by the User management methods
+// It must be called before using those functions.
+func Configure(apiKey string) {
+	DefaultClient.APIKey = apiKey
```

Comment:
Is this setting the client as expected when called?

Reply:
This should be SetAPIKey. I'll add tests to `users_test.go`